README.md

```diff
@@ -86,37 +85,44 @@ the variables are named and described below:
 ### `consul_bin_path`
 
 - Binary installation path
-- Default value: `/usr/local/bin`
+- Default linux value: `/usr/local/bin`
+- Default windows value: `C:\ProgramData\consul\bin`
 
 ### `consul_config_path`
 
 - Base configuration file path
-- Default value: `/etc/consul`
+- Default linux value: `/etc/consul`
+- Default windows value: `C:\ProgramData\consul\config`
 
 ### `consul_configd_path`
 
 - Additional configuration directory
-- Default value: `/etc/consul.d`
+- Default linux value: `/etc/consul.d`
+- Default windows value: `C:\ProgramData\consul\config.d`
 
 ### `consul_data_path`
 
 - Data path
-- Default value: `/var/consul`
+- Default linux value: `/var/consul`
+- Default windows value: `C:\ProgramData\consul\data`
 
 ### `consul_log_path`
 
 - Log path
-- Default value: `/var/log/consul`
+- Default linux value: `/var/log/consul`
+- Default windows value: `C:\ProgramData\consul\log`
 
 ### `consul_run_path`
 
 - Run path for PID file
-- Default value: `/var/run/consul`
+- Default linux value: `/var/run/consul`
+- Default windows value: `C:\ProgramData\consul`
 
 ### `consul_user`
 
 - OS user
-- Default value: *consul*
+- Default linux value: *consul*
+- Default linux value: *LocalSystem*
```

Comment:
Windows convention is to have the `LocalSystem` user run services. It has the right mix of permissions for those kinds of services.